content/en/docs/reference/using-api/api-overview.md

```diff
@@ -18,95 +18,115 @@ This page provides an overview of the Kubernetes API.
+Different API versions indicate different levels of stability and support. You
+can find more information about the criteria for each level in the
+[API Changes documentation](https://git.k8s.io/community/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions).
 
 Here's a summary of each level:
 
 - Alpha:
   - The version names contain `alpha` (for example, `v1alpha1`).
-  - The software may contain bugs. Enabling a feature may expose bugs. A feature may be disabled by default.
+  - The software may contain bugs. Enabling a feature may expose bugs. A
+    feature may be disabled by default.
   - The support for a feature may be dropped at any time without notice.
   - The API may change in incompatible ways in a later software release without notice.
-  - The software is recommended for use only in short-lived testing clusters due to increased risk of bugs
-    and lack of long-term support.
-
+  - The software is recommended for use only in short-lived testing clusters,
+    due to increased risk of bugs and lack of  long-term support.
+  
 - Beta:
   - The version names contain `beta` (for example, `v2beta3`).
-  - The software is well tested. Enabling a feature is considered safe. Features are enabled by default.
+  - The software is well tested. Enabling a feature is considered safe.
+    Features are enabled by default.
   - The support for a feature will not be dropped, though the details may change.
-  - The schema or semantics of objects may change in incompatible ways in a subsequent beta or stable release.
-    When this happens, migration instructions are provided. Schema changes may require deleting, editing,
-    or recreating API objects. The editing process may not be straightforward. Migration may require
-    downtime for applications that rely on the feature.
-  - The software is not recommended for production uses. Subsequent releases may introduce
-    incompatible changes. If you have multiple clusters which can be upgraded independently,
-    you may be able to relax this restriction.
-  - Try the beta features and provide feedback. After the features exit beta, it may not be practical
-    to make more changes.
 
+  - The schema and/or semantics of objects may change in incompatible ways in
+    a subsequent beta or stable release. When this happens, migration
+    instructions are provided. Schema changes may require deleting, editing, and
+    re-creating API objects. The editing process may not be straightforward.
+    The migration may require downtime for applications that rely on the feature.
+  - The software is not recommended for production uses. Subsequent releases
+    may introduce incompatible changes. If you have multiple clusters which
+    can be upgraded independently, you may be able to relax this restriction.
+  
+  {{< note >}}
+  Try the beta features and provide feedback. After the features exit beta, it
```

Comment:
I think I recently removed the note shortcode.
Add it back if you thinks it helps.

Reply:
I've given it a second thought and I'd leave the note shortcode. It really helps in this context.